CONTRIBUTING.md

```diff
@@ -1,9 +1,15 @@
 # Contributors
 
-## How to Contribute
+Thanks for considering making a contribution to the Sui network or its documentation. The Sui monorepo houses the code that powers and documents the Sui blockchain. 
 
-See [Contributing to Sui](doc/src/contribute/index.md) for instructions on helping Sui grow.
+## Contribute to Sui
 
-See [ISSUES.md](./ISSUES.md) for filing bug reports.
+Have an idea for a feature that will improve Sui for everyone? Start with a [Sui Improvement Proposal](https://github.com/sui-foundation/sips/tree/main).
 
-See [CONTRIBUTING.md](./crates/sui-framework/CONTRIBUTING.md) for information related to `sui-framework` crate contributions..
+Found a bug or security vulnerability? Create a [GitHub issue](https://github.com/MystenLabs/sui/issues/new/choose). 
```

Comment:
Question: There is currently nothing about contributing code. Is that intentional?

Reply:
I believe following the SIP process is the recommended approach before contributing code. Your point is valid, though, for smaller code enhancements/fixes. Updated for clarity and addtl info.